api/handler/submit-survey.js

```diff
@@ -20,6 +20,7 @@ function submitSurvey (request, reply) {
     const questionOption = database.sequelize.model('question_option');
     const surveyInstanceId = request.payload.surveyInstanceID;
     const questionInstArr = [];
+    const requestSuccess = 500;
```

Comment:
I am sorry. My mistake.

I have checked the status code check and 500 is wrong.
I am correcting this in the api responses.